spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -4,83 +4,174 @@
 
-      sign_in_and_2fa_user
-      complete_doc_auth_steps_before_agreement_step
-      complete_agreement_step
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+      end
+
+      it 'skips when disabled and redirects to hybrid handoff' do
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
     end
 
-    it 'skips when disabled and redirects to hybird handoff' do
-      expect(page).to have_current_path(idv_hybrid_handoff_url)
+    context 'and when sp has not opted into ipp' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { false }
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+        allow_any_instance_of(ServiceProvider).to receive(:in_person_proofing_enabled).
+          and_return(false)
+        sign_in_and_2fa_user(user, issuer: ipp_service_provider.issuer)
+
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+      end
+
+      it 'skips when disabled and redirects to hybrid handoff' do
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
     end
   end
 
   context 'when both ipp and opt-in ipp are enabled' do
-    before do
-      allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
-      allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+    context 'and when sp has opted into ipp' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+        allow_any_instance_of(ServiceProvider).to receive(:in_person_proofing_enabled).
+          and_return(true)
 
-      sign_in_and_2fa_user
-      complete_doc_auth_steps_before_agreement_step
-      complete_agreement_step
-    end
+        sign_in_and_2fa_user(user, issuer: ipp_service_provider.issuer)
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+      end
 
-    it 'displays expected content and requires a choice' do
-      expect(page).to have_current_path(idv_how_to_verify_path)
+      it 'displays expected content and requires a choice' do
+        expect(page).to have_current_path(idv_how_to_verify_path)
 
-      # Try to continue without an option
-      click_continue
+        # Try to continue without an option
+        click_continue
 
-      expect(page).to have_current_path(idv_how_to_verify_path)
-      expect(page).to have_content(t('errors.doc_auth.how_to_verify_form'))
+        expect(page).to have_current_path(idv_how_to_verify_path)
+        expect(page).to have_content(t('errors.doc_auth.how_to_verify_form'))
 
-      complete_how_to_verify_step(remote: true)
-      expect(page).to have_current_path(idv_hybrid_handoff_url)
+        complete_how_to_verify_step(remote: true)
```

Comment:
we should add another step where remote is `false`. Perhaps instead of an additional test we can go back after line 141 and then redo w/ remote = `false`

Reply:
thanks for adding this additional test!